### PR TITLE
Fixing tx config file

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
Getting this to work for good.

### CODE REVIEW

File: .tx/config [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [main]
 host = https://www.transifex.com
 
-[qgep-project.qgep_ents]
+[o:qgep-project:p:qgep_ents]
```
Comment (m-kuhn) [inline .tx/config:4]: @why-not-try-calmer can you add a short explanation what's being done here / why? Is that a new notation?

Comment (why-not-try-calmer) [reply .tx/config]: Correct. 

- API v.2 (deprecated): `[<project>.<resource>]`
 - API v.3 (latest): `[o:<organization>:p:<project>:r:<resource>]`

**Comment by why-not-try-calmer** [comment]
This should move the repo closer to the solution.

**Comment by why-not-try-calmer** [comment]
Here too @ponceta it would help to be able to rule out the possibility that TX_TOKEN is deprecated.

File: .github/workflows/update-translation.yml [hunk 2]
```diff
@@ -25,6 +25,8 @@ jobs:
           curl -OL https://github.com/transifex/cli/releases/download/v1.3.1/tx-linux-amd64.tar.gz
           tar -xvzf tx-linux-amd64.tar.gz
 
-      - name: package
+      - name: Update translations
+        env:
+          TOK: ${{ secrets.TX_TOKEN }}
         run: |
-          ./scripts/update_and_push_ts_files.sh
+          TX_TOKEN=TOK ./scripts/update_and_push_ts_files.sh
```
[the remark this answers is not in the document]
Comment (why-not-try-calmer) [reply .github/workflows/update-translation.yml:32]: Both were tried and caused an error that was reported slightly differently ("no token found") as opposed to the current one "401".

**Comment by ponceta** [comment]
TX_TOKEN updated (in repository secrets) QGEP / QGEP

**Comment by why-not-try-calmer** [comment]
@ponceta Yesterday we were remembered the hard way that Pull requests from forks (or anything else than `origin`) cannot use secrets from `origin`. Testing thus requires re-opening directly from this repo.

**Pull request closed without merging**